sunpy/io/fits.py

```diff
@@ -103,22 +95,20 @@ def get_header(afile):
     Read a fits file and return just the headers for all HDU's. In each header,
     the key WAVEUNIT denotes the wavelength unit which is used to describe the
     value of the key WAVELNTH.
-
     Parameters
     ----------
     afile : `str` or fits.HDUList
         The file to be read, or HDUList to process.
-
     Returns
     -------
     headers : `list`
         A list of FileHeader headers.
     """
-    if isinstance(afile, fits.HDUList):
+    if isinstance(str(afile), fits.HDUList):
```

Comment:
Why are you casting `afile` into `str` and whether it's an instance of `fits.HDUList`?